doc/introduction/compiling_circuits.rst

```diff
@@ -16,7 +16,7 @@ Compilation functionality is mostly designed as **transforms**, which you can re
 section on :doc:`inspecting circuits </introduction/inspecting_circuits>`.
 
 In addition to quantum circuit transforms, PennyLane also
-supports experimental just-in-time compilation, via
```

Comment:
When dealing with external libraries, how do we decide whether to link to the repository or to the documentation?"

Reply:
I think I am just doing a very rough guess; I'm trying to include the github once, but then after that I only use the docs.